config/boot.rb

```diff
@@ -35,15 +35,15 @@
 # Load Resque configuration and controller
 require 'resque'
 Resque.redis = (defined? REDIS_URL) ? REDIS_URL : "localhost:6379/resque:#{ENV['ROBOT_ENVIRONMENT']}"
-# begin
-#   if defined? REDIS_TIMEOUT
-#     _server, _namespace = REDIS_URL.split('/', 2)
-#     _host, _port, _db = _server.split(':')
-#     _redis = Redis.new(host: _host, port: _port, thread_safe: true, db: _db, timeout: REDIS_TIMEOUT.to_f)
-#     Resque.redis = Redis::Namespace.new(_namespace, redis: _redis)
-#   else
-#     Resque.redis = REDIS_URL
-#   end
-# end
+begin
+  if defined? REDIS_TIMEOUT
+    server, namespace = REDIS_URL.split('/', 2)
+    host, port, db = server.split(':')
+    redis = Redis.new(host: host, port: port, thread_safe: true, db: db, timeout: REDIS_TIMEOUT.to_f)
+    Resque.redis = Redis::Namespace.new(namespace, redis: redis)
+  else
+    Resque.redis = REDIS_URL
```

Comment:
Same feedback.  This `else` clause shouldn't exist.  It should just rely on line 37's assignment, which is smarter and better.  Alternatively, line 37 should be deleted, but it seems to be the preferable version.

Reply:
gotcha.  Removed in commit below.